app/views/contact/govuk/accessible_format_requests/request_sent.html.erb

```diff
@@ -0,0 +1,31 @@
+<% content_for :title do %>
+  <%= t("controllers.contact.govuk.accessible_format_requests.request_sent_heading") %>
+<% end %>
+
+<%= render "govuk_publishing_components/components/panel", title: t("controllers.contact.govuk.accessible_format_requests.request_sent_panel") %>
+
+<%= render "govuk_publishing_components/components/heading", {
+  text: t("controllers.contact.govuk.accessible_format_requests.request_sent_heading"),
+  heading_level: 1,
+  font_size: "l",
+} %>
+
+<p class="govuk-body">
+  <%= raw t("controllers.contact.govuk.accessible_format_requests.request_sent_body") %>
+</p>
+
+<% if content_path %>
+  <p class="govuk-body">
+    <%= link_to t("controllers.contact.govuk.accessible_format_requests.return_to", attachment_title: attachment_title), content_path, class: "govuk-link" %>
+  </p>
+<% end %>
+
+<p class="govuk-body govuk-!-margin-bottom-0">
+  <%= link_to t("controllers.contact.govuk.email_survey_signup.survey_body"), "https://surveys.publishing.service.gov.uk/s/accessible_format/", data: { 
+    track_category: "gem-track-click", 
```

Comment:
@jon-kirwan I've just noticed the test-suite is throwing up an error here - we have track_category twice, once here and once on line 28.

Reply:
Thanks @KludgeKML. I've updated the first one, on line 25, to `track_module: "gem-track-click",`

https://github.com/alphagov/govuk_publishing_components/blob/master/docs/analytics/track-click.md